lambda/es-proxy-layer/lib/supportedLanguages.js

```diff
@@ -246,12 +341,16 @@ const languageErrorMessages = {
         "language": "Tamil",
         "errorMessage": "மன்னிக்கவும், கோரிய மொழி கிடைக்கவில்லை."
     },
+    "te": {
+        "language": "Telugu",
+        "errorMessage": "క్షమించండి, అభ్యర్థించిన భాష అందుబాటులో లేదు."
+    },
     "th": {
         "language": "Thai",
         "errorMessage": "ขออภัย ไม่มีภาษาที่ร้องขอ"
     },
     "tl": {
-        "language": "Tagalog",
+        "language": "Filipino, Tagalog",
```

Comment:
Why the change from "Tagalog" to "Filipino, Tagalog"? Also I think this change would need to be mirrored in the locale-list of codes and names at the top of the file, which it does not?

Reply:
The list of language codes includes both "Tagalog", and "Filipino, Tagalog" - to give more options to set Language.  
In the error message, i used the lanuage name as described in Amazon Translate to be more consistent with the naming. Hence this change in the name. Let me know if this will cause any issues, and can update accordingly. Thanks

Reply:
> Let me know if this will cause any issues
I'm not sure.. I thought the names had to match exactly but maybe not.. If you tested, and it works, all is good.. Thank Raj.